Sources/ResponseValidator/ResponseValidator.swift

```diff
@@ -1,11 +1,27 @@
+// Copyright © 2023 DIAMIR. All Rights Reserved.
+
 import Foundation
 
 /// A type responsible for validating the result produced by a
 /// `URLSession`s `completionHandler` block.
-public protocol ResponseValidator {
+public protocol SyncResponseValidator {
```

Comment:
is the `SyncResponseValidator` still needed? looks the same to me as the normal one